ecommers/lib/core/app_services/product_service.dart

```diff
@@ -12,29 +11,19 @@ class ProductService {
     String searchQuery,
     SortType sortType,
   }) async {
-    final response = await apiService.products(
-      category: category,
-      subCategory: subCategory,
+
+    //TODO: Add category and subCategory type to params
+    return productDataRepository.getProductListByCategory(
```

Comment:
this method uses not only for the filtering by category, so could u please rename it?